opm/simulators/linalg/MILU.cpp

```diff
@@ -223,17 +232,17 @@ void milun_decomposition(const M& A, int n, MILU_VARIANT milu, M& ILU,
         detail::milu0_decomposition ( ILU);
         break;
     case MILU_VARIANT::MILU_2:
-      detail::milu0_decomposition ( ILU, AuxFunctors::identityFunctor(),
-				    AuxFunctors::signFunctor()); 
+      detail::milu0_decomposition ( ILU, identityFunctor<double>(),
```

Comment:
This works as it is only double for now. But the actual type needs to be extracted from the Matrix class, please put
```
using field_type = typename M::field_type;
```
above the switch clause and then use e.g. `identityFunctor<field_type>` to refer to the function. Note: without brack as we are not calling the function but want to pass it.
``